e2e/tests/replacepods/testdata/restart-helper/devspace.yaml

```diff
@@ -4,9 +4,9 @@ name: attach
 pipelines:
   dev:
     run: |-
-      create_deployments test --set helm.values.containers[0].image=ubuntu
-      
-      start_dev test --set imageSelector=ubuntu \
+      create_deployments test --set helm.values.containers[0].image=alpine
```

Comment:
Would it be worthwhile testing both ubuntu & alpine, or are you confident it will work for both?

Reply:
Should be enough using alpine
GNU-userland is a superset of a POSIX one, so if it works in POSIX, it will work on GNU